dockertest/subtestbase.py

```diff
@@ -168,6 +172,21 @@ def failif_not_in(needle, haystack, description=None):
                              (expect_s, description, haystack))
 
     @classmethod
+    def failif_not_redhat(cls, xcept=DockerTestNAError):
+        """
+        Raises an exception if this is a non-RHEL/RHELAH System
+
+        :param xcept: Non-default exception to raise
+        :raise DockerTestNAError: By default
+        """
+        try:
+            with open(cls.redhat_release_filepath, 'rb') as rhrel:
+                if 'Enterprise' not in rhrel.read():
+                    raise xcept("Expecting a Red Hat Enterprise Linux System")
+        except IOError:
+            cls.logwarning("Problem checking /etc/redhat-release")
```

Comment:
Is this really wise? If `/etc/redhat-release` is unreadable/nonexistent/whatever, why would we want to proceed? And if we do proceed, shouldn't the warning at least be courteous enough to offer the reason it failed?

(and, if you keep this code, please use % cls.redhat_release_filepath)